openedx/core/djangoapps/config_model_utils/utils.py

```diff
@@ -21,4 +22,11 @@ def is_in_holdback(user):
         except ExperimentData.DoesNotExist:
             pass
 
+    if enrollment is not None:
+        try:
+            if enrollment.fbeenrollmentexclusion:
+                return True
```

Comment:
Nit: This is inconsistent with the rest of this function, since that code is putting the results of the holdback into `in_holdback`. However, I think this function is probably a reasonable to just use immediate returns (like this code is doing), so it could probably just be cleaned up when you remove the old code.